src/components/dashboard/fleet/charts/PSCDeficienciesChart.jsx

```
@@ -0,0 +1,573 @@
+            <Tooltip content={<PortTooltip />} cursor={{ fill: 'rgba(244, 244, 244, 0.05)' }} />
+            {/* <Legend 
+              layout="horizontal"
+              verticalAlign="top"
+              align="center"
+              wrapperStyle={{
+                paddingBottom: 10
+              }}
+              iconType="circle"
+              iconSize={8}
+            /> */}
+            // Replace your Bar components with this
```

Biome (1.9.4)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)